test/helper/test_trigger.ts

```diff
@@ -581,11 +581,12 @@ const testTriggerFile = (file: string) => {
           const funcStr = func.toString();
           const keys: string[] = [];
 
-          dynamicOutputStringAccess = dynamicOutputStringAccess || /\boutput\[/.test(funcStr);
+          dynamicOutputStringAccess = dynamicOutputStringAccess ||
+            /(?:\boutput\[|\boutput[^.\r\n])/.test(funcStr);
```

Comment:
Added `output`-passed-as-function-argument to this regex, which covers a bunch of other cases that were getting excluded due to bad logic below.